src/main/java/net/logstash/logback/LogstashFormatter.java

```diff
@@ -416,6 +416,22 @@ public void setEnableContextMap(boolean enableContextMap) {
         }
     }
     
+    public String getLineSeparator() {
+        return messageProvider.getLineSeparator();
+    }
+
+    public void setLineSeparator(String lineSeparator) {
+        messageProvider.setLineSeparator(lineSeparator);
+    }
+
+    public boolean isWriteMultiLineMessage() {
+        return messageProvider.isWriteMultiLineMessage();
+    }
+
+    public void setWriteMultiLineMessage(boolean writeMultiLineMessage) {
+        messageProvider.setWriteMultiLineMessage(writeMultiLineMessage);
+    }
```

Comment:
It's not clear that the "lineSeparator" is only used as a regex to split the message into an array if and only if "writeMultiLineMessage" is true.

Additionally, the term "writeMultiLineMessage" does not convey that the message field will be written as an array.  A message with embedded line separators could also be considered a "multi line message".

Perhaps, we can combine these two fields into a single String field named "messageSplitRegex".  When null/empty, the message will not be split, and will be written as a JSON string.  When non-null, the message will be split, and written as a JSON array.